src/components/transactions/TxDetails/TxData/DecodedData/ValueArray/styles.module.css

```diff
@@ -1,8 +1,3 @@
 .nestedWrapper {
   padding-left: 12px;
 }
-
-/* Reduce space between name/address for clarity between rows */
-.address > div {
-  line-height: 1;
-}
```

Comment:
Removed this because it cuts the hanging parts of some letters:
<img width="217" alt="Screenshot 2023-08-21 at 13 03 28" src="https://github.com/safe-global/safe-wallet-web/assets/381895/83dca79d-554b-4b2d-8d13-2010144d8074">